src/lib/utils/preconditions/conditions/IPreconditionCondition.ts

```diff
@@ -15,7 +16,12 @@ export interface IPreconditionCondition {
 	 * @param command The command the message invoked.
 	 * @param entries The containers to run.
 	 */
-	sequential(message: Message, command: Command, entries: readonly IPreconditionContainer[]): PreconditionContainerReturn;
+	sequential(
+		message: Message,
+		command: Command,
+		entries: readonly IPreconditionContainer[],
+		context: PreconditionContext
```

Comment:
Isn't this added required parameter a breaking change? Please document it if so.